silverback-integration-kafka/src/Silverback.Integration.Kafka/Messaging/IDeliveryResultHandler.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Confluent.Kafka;
+
+namespace Silverback.Messaging
+{    
+    /// <summary>
+    /// Delivery Result Handler interface
+    /// </summary>
+    public interface IDeliveryResultHandler : IDisposable
```

Comment:
Why force IDisposable? Couldn't we just check if IDisposable is implemented before calling Dispose?